src/templates/newsTemplate.js

```diff
@@ -26,22 +26,22 @@ export default function Template({
     </Layout>
   )
 }
-export const pageQuery = graphql`
-  query(
-    $path: String!
-    $SQIP_HERO_SHAPE_COUNT: Int
-    $SQIP_THUMB_SHAPE_COUNT: Int
-  ) {
-    markdownRemark(frontmatter: { path: { eq: $path } }) {
-      html
-      frontmatter {
-        date(formatString: "MMMM DD, YYYY")
-        path
-        title
-        featuredImage {
-          ...HeroImage
-        }
-      }
-    }
-  }
-`
+ export const pageQuery = graphql`
```

Comment:
I think this change was unintended.

When making commits it is a best practice to only `git add` the files you intentionally changed.

If it helps, my process is:

- `git status` to check what files have changed.
- `git diff` to see the exact changes and do an initial "code review" pass
- `git add`  with the path to the folder or files you specifically want to be part of the PR. tab completion should help

Reply:
Thanks. I'll get on it as soon as I get a chance!